pyart/aux_io/rainbow_wrl.py

```diff
@@ -0,0 +1,493 @@
+"""
+pyart.aux_io.rainbow
+====================
+
+Routines for reading RAINBOW files (Used by SELEX) using the wradlib library
+
```

Comment:
Add the following lines to the section so that the module function are included in the documentation:
```
.. autosummary::
    :toctree: generated/
```